### PR TITLE
Refactor/ostree and installed versions

### PR DESCRIPTION
**Pull request by lbonn**
Two self-contained commits, part of the manifest-after-reboot work.

Submitted here for more convenient review.

### CODE REVIEW

File: src/libaktualizr/package_manager/debianmanager.cc [hunk 1]
```diff
@@ -54,13 +54,12 @@ data::InstallOutcome DebianManager::install(const Uptane::Target &target) const
...
+  size_t current_k = SIZE_MAX;
+  storage_->loadInstalledVersions(&installed_versions, &current_k);
 
-  std::vector<Uptane::Target>::iterator it;
-  for (it = installed_versions.begin(); it != installed_versions.end(); it++) {
-    if (it->sha256Hash() == current_hash) {
-      return *it;
-    }
+  if (current_k != SIZE_MAX) {
+    return installed_versions.at(current_k);
```
[the remark this answers is not in the document]
Comment (lbonn) [reply src/libaktualizr/package_manager/debianmanager.cc]: Yes, it's usually pretty quick to make changes there.

But we still test the "aktualizr updates itself" in CI! Only in xenial though and it's a bit annoying.

File: tests/ostree-scripts/makephysical.sh [hunk 2]
```diff
@@ -6,26 +6,27 @@ if [ "$#" -lt 1 ]; then
   exit 1
 fi
 
-TARGETDIR=$(realpath $1)
+OSTREE_SYSROOT="$(realpath "$1")"  # replaces `--sysroot=...`
+export OSTREE_SYSROOT
```
[the remark this answers is not in the document]
Comment (lbonn) [reply tests/ostree-scripts/makephysical.sh]: ostree will read the variable for admin commands, so we don't have to supply `--sysroot` every time (really minor I know...).

The export on the second line came from this: https://github.com/koalaman/shellcheck/wiki/SC2155

File: src/libaktualizr/package_manager/debianmanager.cc (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrickvacek) [reply src/libaktualizr/package_manager/debianmanager.cc]: Yeah, we might as well test what we have. It's cool functionality even if nobody is using it. If the CI stuff is super annoying to maintain, we can consider dropping it, though.

File: tests/ostree-scripts/makephysical.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (patrickvacek) [reply tests/ostree-scripts/makephysical.sh]: Nice, good call on both parts.

**Pull request merged**